### PR TITLE
Add documentation for the new met_eireann integration

### PR DESCRIPTION
**Pull request by DylanGore**
## Proposed change

Adds documentation for the new Met Éireann weather integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [https://github.com/home-assistant/core/pull/39429](https://github.com/home-assistant/core/pull/39429)
- Link to parent pull request in the Brands repository: [https://github.com/home-assistant/brands/pull/1820](https://github.com/home-assistant/brands/pull/1820)
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/met_eireann.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<div class="note">
+
+The Met Éireann API will only provide data for Ireland, the UK and a small part of northern France. For specific coverage, please see the Notes on API document available [here](https://data.gov.ie/dataset/met-eireann-weather-forecast-api/resource/027da6d5-d819-48d1-9b16-331dba169bd1).
+
+</div>
+
+## Configuration
+
+To add a Met Éireann weather source, go to __Configuration >> Integrations__ in the UI and create a new Met Éireann entry. You can specify a custom latitude, longitude and elevation if required, if not speciefied the Home Assistant defaults will be used.
```
Comment (klaasnicolaas) [inline source/_integrations/met_eireann.markdown]:
```suggestion
To add a Met Éireann weather source, go to **Configuration** -> **Integrations** in the UI and create a new Met Éireann entry. You can specify a custom latitude, longitude and elevation if required, if not speciefied the Home Assistant defaults will be used.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/met_eireann.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<div class="note">
+
+The Met Éireann API will only provide data for Ireland, the UK and a small part of northern France. For specific coverage, please see the Notes on API document available [here](https://data.gov.ie/dataset/met-eireann-weather-forecast-api/resource/027da6d5-d819-48d1-9b16-331dba169bd1).
+
+</div>
+
+## Configuration
+
+To add a Met Éireann weather source, go to **Configuration** -> **Integrations** in the UI and create a new Met Éireann entry. You can specify a custom latitude, longitude and elevation if required, if not speciefied the Home Assistant defaults will be used.
```
Comment (frenck) [inline source/_integrations/met_eireann.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/met_eireann.markdown [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Data License
+
+The data provied by Met Éireann is licensed under the Met Éireann Open Data Custom License (similar to a Creative Commons CC BY 4.0 licence). A license summary and the full license are available [here](https://data.gov.ie/dataset/met-eireann-weather-forecast-api/resource/027da6d5-d819-48d1-9b16-331dba169bd1). In short, if you distribute, broadcast or make Met Éireann data available on the public internet you must give credit to Met Éireann and display their weather warnings.
```
Comment (frenck) [inline source/_integrations/met_eireann.markdown]:
```suggestion
The data provided by Met Éireann is licensed under the Met Éireann Open Data Custom License (similar to a Creative Commons CC BY 4.0 license). A license summary and the full license are available [here](https://data.gov.ie/dataset/met-eireann-weather-forecast-api/resource/027da6d5-d819-48d1-9b16-331dba169bd1). In short, if you distribute, broadcast or make Met Éireann data available on the public internet you must give credit to Met Éireann and display their weather warnings.
```

File: source/_integrations/met_eireann.markdown [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  - Weather
+ha_release: 2021.5
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@DylanGore'
+ha_domain: met_eireann
+---
+
+The `met_eireann` platform uses the [Met Éireann](https://met.ie) (The Irish Meteorological Service) Public Weather Forecast API to provide current and forecasted weather data for a given location. This integratopn is in no way affiliated with or endoresed by Met Éireann.
```
Comment (frenck) [inline source/_integrations/met_eireann.markdown]:
```suggestion
The Met Éireann integration uses the [Met Éireann](https://met.ie) (The Irish Meteorological Service) Public Weather Forecast API to provide current and forecasted weather data for a given location. This integration is in no way affiliated with or endorsed by Met Éireann.
```

**Comment by frenck** [comment]
Thanks, @DylanGore 👍

**Pull request merged**